orchestra/utils/view_helpers.py

```diff
@@ -12,3 +14,82 @@ class IsAssociatedWorker(permissions.BasePermission):
     def has_object_permission(self, request, view, obj):
         worker = Worker.objects.get(user=request.user)
         return obj.worker == worker
+
+
+def get_changed_fields(old_todo, new_todo):
+    # To avoid Slack noise, only care about title and details
+    changed_fields = []
+    if old_todo.title != new_todo.title:
+        changed_fields.append('title')
+    if old_todo.details != new_todo.details:
+        changed_fields.append('details')
+    if len(changed_fields) > 1:
+        changed_fields[-1] = 'and {}'.format(changed_fields[-1])
+    return ', '.join(changed_fields)
+
+
+def get_relevance_and_completion_changes(old_todo, new_todo):
+    # When activity_log is updated, `todo_change = ''`
+    # to avoid triggering any slack messages
+    todo_change = ''
+    if old_todo.completed != new_todo.completed:
+        todo_change = 'complete' if new_todo.completed else 'incomplete'
+    elif old_todo.skipped_datetime != new_todo.skipped_datetime:
+        todo_change = 'not relevant' \
+            if new_todo.skipped_datetime else 'relevant'
+    return todo_change
+
+
+def get_update_message(old_todo, new_todo, sender=None):
+    message = ''
+    changes = get_relevance_and_completion_changes(old_todo, new_todo)
+    changed_fields = get_changed_fields(old_todo, new_todo)
+    head = '{} has updated `{}`:'.format(sender, new_todo.title) \
+        if sender else '`{}` has been updated:'.format(new_todo.title)
+    body = 'marked as {}'.format(changes) if changes else ''
```

Comment:
Sorry again about tests :/

```suggestion
    body = 'marked {}'.format(changes) if changes else ''
```